sorc/build_fv3.sh

```diff
@@ -22,11 +22,11 @@ if [ $target = orion ]; then target=orion.intel ; fi
 cd fv3gfs.fd/
 FV3=$( pwd -P )/FV3
 cd tests/
-
-if [ ${RUN_CCPP:-${1:-"NO"}} = "NO" ]; then
+mkdir ../NEMS/exe
```

Comment:
This errors out if the folder exists. Please change this line to:

```
if [ ! -d ../NEMS/exe ]; then mkdir ../NEMS/exe ; fi
```

Thanks!